resources/views/editTemplateView.blade.php

```diff
@@ -0,0 +1,220 @@
+@extends('layouts.master')
+
+@section('title')
+    {{ trans('mainLang.editDetails') }}
+@stop
+
+@section('content')
+    @if($template->id == null)
+        {!! Form::open(['method' => 'POST', 'route' => ['template.update', 0], 'class'=> 'form-inline']) !!}
+    @else
+        {!! Form::open(['method' => 'POST', 'route' => ['template.update', $template->id], 'class'=> 'form-inline']) !!}
+    @endif
+    <div class="row">
+        <div class="panel col-md-6 col-sm-12 col-xs-12 no-padding">
+            <div class="panel-heading">
+                <h4 class="panel-title">{{ trans('mainLang.changeEventJob') }}:</h4>
+            </div>
+
+            <br>
+
+            <div class="panel-body no-padding">
```

Comment:
Padding broken here, bottom left corner, probably need to remove the no-padding class or add another styling.